data/event-schemas/schema/m.secret.send.yaml

```
@@ -0,0 +1,32 @@
+---
+allOf:
+  - $ref: core-event-schema/event.yaml
+description: |-
+  Sent by a client to share a secret with another device, in response to an
+  `m.secret.request` event. It must be encrypted as an `m.room.encrypted` event
+  using [Olm](#molmv1curve25519-aes-sha2), then sent as a to-device event.
+
+  The `request_id` must match the ID previously given in an `m.secret.request`
+  event. The recipient must ensure that this event comes from a device that the
+  `m.secret.request` event was originally sent to, and that the device is
+  trusted. This should be done by checking the sender key of the Olm session that
+  the event was sent over.
```

```suggestion
  The `request_id` must match the ID previously given in an `m.secret.request`
  event. The recipient must ensure that this event comes from a device that the
  `m.secret.request` event was originally sent to, and that the device is
  a trusted device owned by the recipient. This should be done by checking the
  sender key of the Olm session that the event was sent over.
```

We should only accept secrets from our own devices.